ArgumentValidator.php

```diff
@@ -153,4 +165,26 @@ private function validateClass($expectedClassName, $actualClassName)
             ));
         }
     }
+
+    /**
+     * Checks if the installed ExpressionLanguage component version is supported
+     * by the installed version of the DependencyInjection component.
+     *
+     * Starting with Symfony 2.6, the DependencyInjection component requires the
+     * ExpressionLanguage component to be of version 2.6 or higher.
+     *
+     * @return bool
+     */
+    private function isExpressionLanguageSupported()
+    {
+        if (!method_exists('Symfony\Component\DependencyInjection\Definition', 'getFactory')) {
```

Comment:
Well, to be more precise, enabling expression validation in the `Configuration` class should maybe check if the ExpressionLanguage component is really present.